src/main/java/frc/robot/Constants.java

```diff
@@ -152,8 +154,9 @@ public static final class ModuleConstants {
     public static final double kDrivingMinOutput = -1.0;
     public static final double kDrivingMaxOutput = 1.0;
 
+    //was 3.75,0,.35
     public static final PIDGains kTurningPIDGains =
-        new PIDGains(4, 0, 0.075); // TODO: tune values for the turning motor
+        new PIDGains(6.75, 0.03, 0.35); // TODO: tune values for the turning motor
```

Comment:
Are these tuned values good now?  Can we remove the TODO?

Reply:
No they need to be retuned with the feedforward